Source/WebCore/platform/graphics/BitmapImage.h

```diff
@@ -76,6 +76,9 @@ class BitmapImage final : public Image {
 #if ASSERT_ENABLED
     bool hasSolidColor() final { return m_source->hasSolidColor(); }
 #endif
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+    bool shouldUseQuickLookForFullscreen() const final { return m_source->shouldUseQuickLookForFullscreen(); }
+#endif
```

Comment:
I think this can be private (at least for this patch) since you are using it through the `Image` not the `BitmapImage`.